openmetadata-ui/src/main/resources/ui/src/components/AppRouter/AuthenticatedAppRouter.tsx

```diff
@@ -341,12 +351,6 @@ const AuthenticatedAppRouter: FunctionComponent = () => {
       <Route exact component={SwaggerPage} path={ROUTES.SWAGGER} />
       <Route exact component={DomainVersionPage} path={ROUTES.DOMAIN_VERSION} />
 
-      <Route
-        exact
-        component={ServiceVersionPage}
-        path={ROUTES.SERVICE_VERSION}
-      />
```

Comment:
Why did we remove this?

Reply:
We didn't remove it , just relocated it above the service details page. Since I introduced sub tab in service details page, it was not routing to the version pages